Lib/test/clinic.test.c

```diff
@@ -1675,17 +1685,65 @@ test_Py_ssize_t_converter(PyObject *module, PyObject *const *args, Py_ssize_t na
     if (!_Py_convert_optional_to_ssize_t(args[2], &c)) {
         goto exit;
     }
+    if (nargs < 4) {
+        goto skip_optional;
+    }
+    {
+        Py_ssize_t ival = -1;
+        PyObject *iobj = _PyNumber_Index(args[3]);
+        if (iobj != NULL) {
+            ival = PyLong_AsSsize_t(iobj);
+            Py_DECREF(iobj);
+        }
+        if (ival == -1 && PyErr_Occurred()) {
+            goto exit;
+        }
+        d = ival;
+        if (d < 0) {
+            PyErr_SetString(PyExc_ValueError,
+                            "d must be >=0");
```

Comment:
I'm not a fan of `>= 0` here, I'd prefer to use words (must be at least 0 / must not be a negative number / must be non-negative). Especially for a common error message we should being overly mathematical, I think.

A

Reply:
I've adjusted to 'cannot be negative', as it's the same length and more widely used.